### DISCUSSION TITLE
Support of multiple accessorKey for global search `mantine-data-grid`

### DISCUSSION THREAD

**Discussion by dibdutta** [mantinedev/mantine#2456] [Feature requests]
I wanted to have support of multiple accessorKey to enable search by multiple fields , especially for nested DOM column.
Also if `mantine-data-grid` supports hidden column, that also can be an workaround

**Comment by rtivital** [comment]
Hi @dibdutta, Mantine does not provide data grid component, you are using a third-party library which is not managed by Mantine maintainers, you should forward your request and issues to the maintainers of that library.

**Reply by dibdutta** [reply]
Thank you , I will close the request
